src/cheri/zylevelsN.adoc

```diff
@@ -0,0 +1,143 @@
+[#section_zylevelsN,reftext="ZylevelsN"]
+== "ZylevelsN" Extension for CHERI Multi-level Information Flow Control
```

Comment:
Thanks for keeping this file, I agree it should not be part of the spec but it's useful to keep around.